docs/user-guide/creating-application/app-metrics.md

```diff
@@ -50,6 +50,14 @@ This metrics indicates the  application’s response to client’s request with
 
 Latency metrics shows the latency for an application. Latency measures the delay between an action and a response.
 
+**99.9th percentile latency**: The maximum latency, in seconds, for the fastest 99.9% of requests. By default the value is 99.9% percentile.
```

Comment:
Remove `By default the value is 99.9% percentile.` It's clear form the UI itself